elixir/apps/web/lib/web/live/temp_accounts/index.ex

```diff
@@ -0,0 +1,211 @@
+defmodule Web.TempAccounts.Index do
+  use Web, {:live_view, layout: {Web.Layouts, :public}}
+  alias Domain.{Accounts, Actors, Auth, Config}
+
+  def mount(_params, _session, socket) do
+    if Config.global_feature_enabled?(:temp_accounts) do
+      socket =
+        assign(socket,
+          page_title: "Try Firezone",
+          account_info: nil,
+          creation_error: false
+        )
+
+      {:ok, socket}
+    else
+      raise(Web.LiveErrors.NotFoundError)
+    end
+  end
+
+  def render(assigns) do
+    ~H"""
+    <section>
+      <div class="flex flex-col items-center justify-center px-6 py-8 mx-auto lg:py-0">
+        <.hero_logo text="Welcome to Firezone" />
+
+        <div class="w-full col-span-6 mx-auto bg-white rounded shadow md:mt-0 sm:max-w-lg xl:p-0">
+          <div class="p-6 space-y-4 lg:space-y-6 sm:p-8">
+            <.flash flash={@flash} kind={:error} />
+            <.flash flash={@flash} kind={:info} />
+            <.welcome :if={is_nil(@account_info) and @creation_error == false} />
+            <.account
+              :if={not is_nil(@account_info)}
+              account={@account_info.account}
+              password={@account_info.password}
+            />
+            <div :if={@creation_error}>
+              Something went wrong!
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    """
+  end
+
+  def welcome(assigns) do
+    ~H"""
+    <div class="text-center">
+      <p class="mb-4">
+        Interested in trying out Firezone?  You've come to the right place!
+      </p>
+
+      <p class="mb-4">
+        Take Firezone for a spin with a temporary demo account.
+      </p>
+      <div class="py-1">
+        <.button class="w-full" phx-click="start">Create Temporary Account</.button>
+      </div>
+
+      <div class="flex items-center my-6">
+        <div class="w-full h-0.5 bg-neutral-200"></div>
+        <div class="px-5 text-center text-neutral-500">or</div>
+        <div class="w-full h-0.5 bg-neutral-200"></div>
+      </div>
+
+      <p class="mb-4">
+        Sign up below with a free starter account to keep your data.
+      </p>
+      <p>
+        <a
+          class="inline-block bg-white border rounded px-3 py-2 text-sm w-full"
+          href={url(~p"/sign_up")}
+        >
+          Create Free Starter Account
+        </a>
+      </p>
+    </div>
+    """
+  end
+
+  attr :account, :any
+  attr :password, :string
+
+  def account(assigns) do
+    ~H"""
+    <div class="space-y-6">
+      <div class="text-center text-neutral-900 text-xl">
+        Your temporary account has been created!
+      </div>
+      <.flash kind={:error}>
```

Comment:
```suggestion
      <.flash kind={:warning}>
```

Do we have a warning color? Just thinking it might be a little better to keep them trained on red for actual errors

Reply:
Yep, good call.  Updated.